velox/substrait/TypeUtils.cpp

```diff
@@ -18,7 +18,7 @@
 
 namespace facebook::velox::substrait {
 namespace {
-std::vector<std::string_view> getRowTypesFromCompoundName(
+std::vector<std::string_view> getTypesFromCompoundName(
```

Comment:
The comment inside this function needs updating as well. It currently says "CompoundName is like ROW<BIGINT,DOUBLE>..."